src/tss2-tcti/tcti-spi.c

```diff
@@ -0,0 +1,612 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2020 Fraunhofer SIT. All rights reserved.
+ */
+#include <errno.h>
+#include <fcntl.h>
+#include <inttypes.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
+#include <assert.h>
+
+#include "tss2_tcti.h"
+#include "tss2_tcti_spi.h"
+#include "tss2_mu.h"
+#include "tcti-common.h"
+#include "tcti-spi.h"
+#include "util/io.h"
+#define LOGMODULE tcti
+#include "util/log.h"
+
+static void spi_tpm_delay_ms(TSS2_TCTI_SPI_CONTEXT* ctx, int milliseconds)
```

Comment:
mark these helper functions as inline?
Code is probably identical, but still

Reply:
I figured gcc would be smart enough to automatically inline, but you're right and `spi_tpm_read_be32` was also explicitly marked as inline. So all platform helper methods will be inlined with the next force-push.